scripts/bash_lib.sh

```diff
@@ -437,22 +439,40 @@ upgrade_test_type() {
   esac
 }
 
-get_columnstore_logs() {
+save_failure_logs() {
+  local logdir=/home/buildbot/logs/
```

Comment:
can't we use relative path? this has proven to be fragile on mac freebsd and other OS. I know we don't have install/upgrade test there but relative is better if possible since it makes it possible to have both dev and prod builders on the same machine...

Reply:
which relative path `../logs`?

Reply:
to be tested but yes, probably